package.json

```diff
@@ -12,47 +12,46 @@
     ]
   },
   "dependencies": {
-    "@auth0/auth0-react": "^1.3.0",
```

Comment:
Plasmic wanted to update some dependencies so I decided to go ahead and bring everything up to the latest.